components/src/core/Drawer/DrawerNavGroup/DrawerNavGroup.tsx

```diff
@@ -290,6 +303,8 @@ const DrawerNavGroupRender: React.ForwardRefRenderFunction<unknown, DrawerNavGro
                                 statusColor={railItem.statusColor}
                                 title={railItem.title}
                                 disableRailTooltip={railItem.disableRailTooltip}
+                                sx={railItem.sx}
+                                style={railItem.style}
```

Comment:
297-307: it's strange that we are calling out all of these properties individually (error prone and not very future proof). Can't we just take those out and uncomment lioe 284 to spread all the props?

Reply:
Agreed that this is weird, most likely just following the pattern that's been there, but I'm happy to clean this up.

Reply:
I _think_ I may have been the one who originally commented that spread out and I think it had something to do with spreading some of the props that are available in DrawerNavItem but not in DrawerRailItem. So my suggestion might not be valid...might need to test that specific case to see if the spread causes any issues.

Reply:
After some testing, this works fine for the standard rail items but not those passed in via the items prop on `DrawerNavGroup`. I'm going to leave this as is for now.